packages/engine/README.md

```diff
@@ -109,10 +109,20 @@ Depending on your needs, different dependencies are required. Building this proj
 
 ### macOS Developer Specific Instructions
 
-Due to ARM-Based Macs, the `macos` `target_os` has some added complications for development.
+#### For all macs
+
+Unfortunately, Apple currently doesn't provide a way to resize shared-memory allocations. To work around this allocations need to be sufficiently big such that they will not need to be resized. This can be done by setting the `OS_MEMORY_ALLOC_OVERRIDE` environment variable. Setting it to `250000000` should suffice. This can be done using the command line
+
+```sh
+export OS_MEMORY_ALLOC_OVERRIDE=250000000
```

Comment:
```suggestion
Unfortunately, Apple currently doesn't provide a way to resize shared-memory allocations. To work around this allocations need to be sufficiently big such that they will not need to be resized. This can be done by setting the `OS_MEMORY_ALLOC_OVERRIDE` environment variable. A reasonable starting value might be `250000000`, but it is heavily dependent on the memory-requirements of your simulation. This can be done using the command line

```sh
export OS_MEMORY_ALLOC_OVERRIDE=250000000
```